server/src/test/resources/csars/yaml/valid/lamp-csar_input/template.yaml

```diff
@@ -0,0 +1,83 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+description: Template for deploying a LAMP stack.
+metadata:
+  template_name: lamp-stack-template
+  template_author: stupro-toscana
+  template_version: 1.0
+
+
+topology_template:
+  inputs:
+    database_name:
+      type: string
+    database_port:
+      type: integer
+    database_password:
+      type: string
+
+  node_templates:
+    my_app:
+      type: tosca.nodes.WebApplication
+      requirements:
+        - host: apache_web_server
+        - database_endpoint: my_db
+      interfaces:
+        Standard:
+          inputs: database_host: { get_attribute: [ server, private_address ] }
+          create: my_app/install_php.sh
+          start:
+            implementation:
+              primary: my_app/start_myphpapp.sh
+              dependencies:
+                - myphpapp.php
+                - mysql-credentials.php
+
+    apache_web_server:
+      type: tosca.nodes.WebServer.Apache
+      requirements:
+        - host: server
+
+    my_db:
+      type: tosca.nodes.Database.MySQL
+      properties:
+        name: { get_input: database_name }
+        password: { get_input: database_password }
+        port: { get_input: database_port }
+      capabilities:
+        database_endpoint:
+          properties:
+            port: { get_property: [ SELF, port ] }
+      requirements:
+        - host: mysql_dbms
+
+    mysql_dbms:
+      type: tosca.nodes.DBMS.MySQL
+      properties:
+          root_password: { get_input: database_password }
+          port: { get_input: database_port }
+        requirements:
+          - host: server
+        interfaces:
+          Standard:
+            inputs:
+              db_root_password: { get_property: [ SELF, root_password ] }
+            configure: mysql_dbms/mysql_dbms_configure.sh
+
+    server:
+      type: tosca.nodes.Compute
+      capabilities:
+        host:
+          properties:
+            num_cpus: 1
+            disk_size: 25 GB
+            mem_size: 2048 MB
+        os:
+          properties:
+            type: linux
+            distribution: ubuntu
+            version: 16.04
+
+    outputs:
+      server_url:
+        description: Concatenated URL with public server address and port
+        value: { concat: [ 'http://', get_attribute: [ server, public_address ], ':', get_attribute: [ server, port ]] }
```

Comment:
We probably won't support the concat function in the near future. I suggest using two different outputs for now: server_ip and server_port